src/modules/likes.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable operator-linebreak */
+const BASE_URL =
```

Comment:
- [ ] Same here, Please remove this linters disablement line and fix all eslint issues yourself.